whipper/command/mblookup.py

```diff
@@ -1,46 +1,57 @@
 from whipper.command.basecommand import BaseCommand
-from whipper.common.mbngs import musicbrainz
+from whipper.common.mbngs import musicbrainz, getReleaseMetadata
 
 
 class MBLookup(BaseCommand):
     summary = "lookup MusicBrainz entry"
-    description = """Look up a MusicBrainz disc id and output information.
+    description = """
+Look up a MusicBrainz disc id or release id and output information.
 
 You can get the MusicBrainz disc id with whipper cd info.
 
-Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-"""
+Example disc id: KnpGsLhvH.lPrNc1PBL21lb9Bg4-
+"""
 
     def add_arguments(self):
-        self.parser.add_argument(
-            'mbdiscid', action='store', help="MB disc id to look up"
-        )
+        self.parser.add_argument('-d', '--disc-id',
+                                 action='store', dest='disc_id',
+                                 help="MB disc id to look up", default=None)
+        self.parser.add_argument('-R', '--release-id',
+                                 action='store', dest='release_id',
+                                 help="MB release id to look up", default=None)
```

Comment:
Would it be possible to make it so you can just paste an ID and the function will determine whether it’s a Disc ID or an MBID? (So that the commandline command remains the same.)

I’m not sure if that’d be a good idea though, just a thought while looking over this.

Reply:
I gave this some thoughts and IMHO, it might make whipper depends on how the `id`(s) in Musicbrainz is created . Those might be changed (even tho it is not likely) and `mblookup` will not work.

Reply:
> Those might be changed

They won’t. Lots and lots of things rely on MBIDs being UUIDs and Disc IDs being… whatever they are. Neither will change. Maybe something new will be introduced, but those two are here to stay as-is.

Reply:
I get it. Should i implement it right away?

Reply:
Depends on what @JoeLametta and/or @MerlijnWajer thinks.

Reply:
It depends: if the output of the two options gives the same kind of information and it's going to stay this way, I think merging them is OK, otherwise this idea probably needs more thought.

Reply:
@JoeLametta , so you mean that it should be implemented, am i right ?

Reply:
> Hi @JoeLametta , so you mean that it should be implemented, am i right ?

I mean: does the output of the two options look the same right now? If yes, and keeping it this way even in the future makes sense, then I would say go ahead and merge them. Otherwise (different output structure / different output needs for the future) we need to discuss this change with more detail.

BTW: English isn't my native tongue, hopefully I managed to explain myself.

Reply:
Yes me too, i thought merging was "merge" two options together :D. Thanks for your review.

Reply:
> Yes me too, i thought merging was "merge" two options together :D.

Sorry, I could have worded it better but that's exactly what I meant: unifying those two options in case of an affirmative reply to this question :wink:

> does the output of the two options look the same right now? If yes, and keeping it this way even in the future makes sense, then I would say go ahead and merge them.

Reply:
So, it is a "yes" and you agree with @Freso 's suggestion.

Reply:
@ABCbum Sorry, could you please reply to this question first?

> does the output (printed information to the user) of the two options look the same right now (with the current code of this pull request)?

Reply:
Ah yes it does

Reply:
No problem. :+1: 
Last question: do you think the current output is fine or it may be necessary to differentiate the output of the two options?

Reply:
I think unifying it is a little more convinient to users. (i'm not sure)

Reply:
OK, green light from me to the unification of the two options. Don't know what's @MerlijnWajer's opinion about this...

Reply:
But also adding options i think clarifies to users a little bit more about the mblookup's command funcitonalities(what it is capable of, i mean i can only lookup based on release id and disc id now)

Reply:
(FWIW (just to clarify), I’m not **for** or **against** my own idea per se. It was just a thought I had. I can see both pros and cons of it and was mostly thought of as a way to maintain backwards CLI "API" compatibility, which is really a non-issue since we’re still on version <1.0.0.)

Reply:
I have made new commits to unify two options.